src/Microsoft.Data.Sqlite.Core/Extensions/SQLitePCLExtensions.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace SQLitePCL
+{
+    internal static class SQLitePCLExtensions
+    {
+        public static void Dispose2(this sqlite3 db)
```

Comment:
Underneath, we switched from calling `sqlite3_close` to `sqlite3_close_v2`. *Dispose2* seemed appropriate 😉